packages/components/src/combobox-control/style.scss

```diff
@@ -0,0 +1,70 @@
+.components-combobox-control {
+	color: $dark-gray-500;
+	position: relative;
+}
+
+.components-combobox-control__label {
+	display: block;
+	margin-bottom: 5px;
+}
+
+.components-combobox-control__button {
+	border: 1px solid $dark-gray-200;
+	border-radius: 4px;
+	color: $dark-gray-500;
+	display: inline-block;
+	min-height: 30px;
+	min-width: 130px;
+	position: relative;
+	text-align: left;
+
+	&:focus {
+		border-color: $blue-medium-500;
+	}
+
+	&-input {
+		border: none;
+		height: calc(100% - 2px);
+		left: 1px;
+		padding: 0 4px;
+		position: absolute;
```

Comment:
Removing this `position: absolute` doesn't seem to break layout. What's the reasoning behind it?

Reply:
It does break the layout. The styles are being included twice for some reason, and you probably only disabled it in the first set.